### DISCUSSION TITLE
扩容后的固件使用值守式更新后，重启后引导丢失

### DISCUSSION THREAD

**Discussion by iulo** [immortalwrt/immortalwrt#1549] [Q&A] [answered]
原始系统是 `23.05.2-x86-64-generic-squashfs-combined-efi`，使用parted工具本地扩容后刷入x86物理设备。使用值守式更新重启后，发现卡在bios，查看boot选项发现无引导项了

想请问，是否这种手动扩容后的固件无法再使用引导式更新

**Comment by OiCkilL** [comment]
我也遇上一样的问题，于是我把镜像下下来发现生成的镜像是传统bios的而不是efi镜像。所以没法正常引导

**Reply by 1715173329** [reply, marked as answer]
luci 的 bug，最新的 23.05 快照版本已经修好了。

**Reply by OiCkilL** [reply]
> luci 的 bug，最新的 23.05 快照版本已经修好了。

这样啊。不想用快照版本，那我还是等23.05.5再更了
